include/client/networkclient.h

```diff
@@ -135,3 +135,5 @@ public slots:
 
 }  // namespace client
 }  // namespace veles
+
+Q_DECLARE_METATYPE(veles::client::NetworkClient::ConnectionStatus)
```

Comment:
To use queued signal-slot connections with veles::client::NetworkClient::ConnectionStatus as a parameter.